docs/application/dotnet/guides/nui/nui-components/Toast.md

```diff
@@ -38,6 +38,90 @@ Following output is generated when the toast is created using property:
 
 ![Toast](./media/toast.gif)
 
+## Create with style
+
+To create a toast using style, follow these steps:
+
+1. Create a style for toast:
+
+    ```cs
+    ToastStyle style = new ToastStyle
+    {
+        Size = new Size(512, 132),
+        Text = new TextLabelStyle
+        {
+            Padding = new Extents(96, 96, 38, 38),
+            PointSize = 26,
+            TextColor = Color.White,
+        },
+        BackgroundImage = "Poptoast_background.png",
+        BackgroundImageBorder = new Rectangle(64, 64, 4, 4),
+        Duration = 1500
+    };
+    ```
+
+2. Use the style to create a toast and add it to parent:
+
+    ```cs
+    utilityBasicToast = new Toast(style);
+    utilityBasicToast.TextArray = new string[1] { "null parameter" };
+    utilityBasicToast.Position = new Position(650, 350);
+    root.Add(utilityBasicToast);
+    ```
+
+Following output is generated when the toast is created using property:

```

Comment:
Following output is generated when the toast is created using _property_:

Shouldn't  "style" be used here instead of "property"?

Reply:
@mijin85cho 
Thank you for pointing this.
I have missed this.
It should be "style".